From f5b156f417d172c398902206f3eb3832a1c668b2 Mon Sep 17 00:00:00 2001
From: Suneel Garapati <sgarapati@marvell.com>
Date: Fri, 5 Jun 2020 15:16:23 -0700
Subject: [PATCH 0850/1921] drivers: spi: octeontx2: Include SUBSID for T106
 platform

Add subsystem ID for T106 to enable enumeration.

Change-Id: I85c5db717c5e3f55b854f0f0f33450293fd88656
Signed-off-by: Suneel Garapati <sgarapati@marvell.com>
Reviewed-on: https://sj1git1.cavium.com/c/IP/SW/kernel/linux/+/30385
Reviewed-by: Chandrakala Chavva <cchavva@marvell.com>
Tested-by: Sunil Kovvuri Goutham <Sunil.Goutham@cavium.com>
Signed-off-by: Sunil Goutham <sgoutham@marvell.com>
[WK: The original patch got from Marvell sdk11.21.09]
Signed-off-by: Wenlin Kang <wenlin.kang@windriver.com>
---
 drivers/spi/spi-octeontx2.c | 3 +--
 drivers/spi/spi-octeontx2.h | 2 +-
 2 files changed, 2 insertions(+), 3 deletions(-)

diff --git a/drivers/spi/spi-octeontx2.c b/drivers/spi/spi-octeontx2.c
index eab09894bc6c..24a1406e5161 100644
--- a/drivers/spi/spi-octeontx2.c
+++ b/drivers/spi/spi-octeontx2.c
@@ -365,8 +365,7 @@ static const struct pci_device_id octeontx2_spi_pci_id_table[] = {
 	{ PCI_DEVICE_SUB(PCI_VENDOR_ID_CAVIUM,
 			 PCI_DEVID_OCTEONTX2_SPI,
 			 PCI_VENDOR_ID_CAVIUM,
-			 PCI_SUBSYS_DEVID_OTX3) },
-
+			 PCI_SUBSYS_DEVID_CN10K_A) },
 	{ 0, }
 };
 
diff --git a/drivers/spi/spi-octeontx2.h b/drivers/spi/spi-octeontx2.h
index d148a62a65e1..9038d052c4bc 100644
--- a/drivers/spi/spi-octeontx2.h
+++ b/drivers/spi/spi-octeontx2.h
@@ -10,7 +10,7 @@
 #define PCI_SUBSYS_DEVID_OTX2_95XX 0xB300
 #define PCI_SUBSYS_DEVID_OTX2_LOKI 0xB400
 #define PCI_SUBSYS_DEVID_OTX2_95MM 0xB500
-#define PCI_SUBSYS_DEVID_OTX3 0xB900
+#define PCI_SUBSYS_DEVID_CN10K_A 0xB900
 
 #define OCTEONTX2_SPI_MAX_BYTES 1024
 #define OCTEONTX2_SPI_MAX_CLOCK_HZ 25000000
-- 
2.31.1

